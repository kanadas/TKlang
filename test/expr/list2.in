//void_list = void * char_list + void
let repeat = (\n -> if n == 0 then 2@() else 1@((), repeat (n-1))) in repeat 5
