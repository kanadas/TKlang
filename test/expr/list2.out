expr/list2.in
Unbound variable repeat
